cudapoa/src/cudapoa_kernels.cu

```diff
@@ -331,6 +333,12 @@ __global__ void generatePOAKernel(uint8_t* consensus_d,
         }
 
         __syncwarp();
+
+        warp_error = __shfl_sync(0xffffffff, warp_error, 0);
```

Comment:
Use `__shfl_sync(FULL_MASK, warp_error, 0)` for better readability. `FULL_MASK` is defined in `cudapoa_kernels.cuh`. 
Also, make the same change to line 217.

Reply:
Absolutely, done!